js/src/setup-ads/ads-stepper/index.js

```diff
@@ -10,7 +10,6 @@ import { useState } from '@wordpress/element';
  */
 import SetupAccounts from './setup-accounts';
 import AdsCampaign from '.~/components/paid-ads/ads-campaign';
-import SetupBilling from './setup-billing';
```

Comment:
@kt-12 Looks like the `SetupBilling` component is no more used in the codebase. Can you remove the components and associated unused functions/components please?

Reply:
Removed.